### ISSUE TITLE
ETABS_Toolkit: Implementation of GeneralisedTSectionProfile

### ISSUE THREAD

**Issue by kThorsager** [BHoM/ETABS_Toolkit#214]
#### Description:
Implementation of `GeneralisedTSectionProfile`
From the looks of it ETABS does not have any standard sections which can handle BHoM's `GeneralisedTSectionProfile` so it would have to be via the "Section Designer" in ETABS. 

I could not find any proper method for assigning a section in the Section Designer via API but I did not look very hard...

If that is impossible it would at least be nice to get the data pushed in some manner (like, assigning a false section where the notes define how the section should be) so that it would be easier to both find the faulty section and find the real section properties to assign it

related to #213

**Comment by JosefTaylor** [comment]
Correct, section designer is not yet in the API. Not sure how we're going to get that going. 

If the section has already been defined in ETABS (using section designer) then a section with the same name will point to it.

Here are our choices for how to push unsupported bars to ETABS as I see them:

1. Create an explicit section (general section in ETABS) and throw a reflection note
2. Create a null section and throw a reflection warning
3. Create a 1m diameter rod and throw a reflection warning
4. Don't send the bar, throw a reflection error

**Comment by kThorsager** [comment]
A note to that is that asymmetric sections (tex `GeneralisedTSectionProfile`) does not get its J calculated (at least in my case, got a reflection error`1. Can only calculate torsional constant of symmetric T sections or angles`), which is needed in General section definition
Perhaps that can be added

Barring that I like no 3 as it seems like easiest to continue work from, as you can still find your bar in ETABS and "simply" reassign the section
